utilities/calculations.js

```diff
@@ -0,0 +1,64 @@
+export const calculations = {
+  getPercentages(activities, type = '') {
+    
+    const activityTotals = activities.map(activity => {
+      return activity.minutes
+    })
+
+    const totalMinsAllActivities = activityTotals.reduce((total, activity) => {
+      return total + activity
+    }, 0)
+
+    const compiledActivityData = activityTotals.map((total, i) => {
+      return {
+        name: activities[i].activityName,
+        percentage: Math.round(total / totalMinsAllActivities * 100),
+        moduleId: activities[i].moduleId
+      }
+    })
+
+    if (type === 'activity') {
+      return compiledActivityData.reduce((acc, curr) => {
+        if (acc[curr.name]) {
+          acc[curr.name] += curr.percentage
+
+        } else {
+          acc[curr.name] = curr.percentage
+        }
+
+        return acc
+      }, {})
+
+    } else {
+      return compiledActivityData
+    }
+  }
+}
+
+    // getActivityMinTotal(activity) {
```

Comment:
If Lucas doesn't need these functions for his chart data then we should be all good to delete these comments

Reply:
Removed!